docs/reference/metricbeat/metricbeat-module-gcp.md

```diff
@@ -36,6 +36,9 @@ This is a list of the possible module parameters you can tune:
 * **exclude_labels**: (`true`/`false` default `false`) Do not extract extra labels and metadata information from metricsets and fetch metrics only. At the moment, **labels and metadata extraction is only supported** in `compute` metricset.
 * **period**: A single time duration specified for this module collection frequency.
 * **endpoint**: A custom endpoint to use for the GCP API calls. If not specified, the default endpoint will be used.
+* **metadata_cache**: (`true`/`false` default `false`) Enable caching of metadata. If set to true, metadata will be cached to improve performance.
+* **metadata_cache_refresh_period**: A duration specifying how often the cached metadata should be refreshed (e.g., `5m`, `1h`).
```

Comment:
Going forward we should be labeling new features with the version number it was added in. I'm working on some updated contributing guidelines, but for now this should suffice:

```suggestion
* **metadata_cache**: (`true`/`false` default `false`) Enable caching of metadata. If set to true, metadata will be cached to improve performance. {applies_to}`product: ga 9.1.0`
* **metadata_cache_refresh_period**: A duration specifying how often the cached metadata should be refreshed (e.g., `5m`, `1h`). {applies_to}`product: ga 9.1.0`
```

We're actively working on improving the rendered version badges, but for now this is what it will look like:

<img width="841" alt="image" src="https://github.com/user-attachments/assets/ab273249-2b21-4fac-84c9-b57415578dbe" />